driver-benchmarks/src/main/com/mongodb/benchmark/benchmarks/GridFSExportBenchmark.java

```diff
@@ -0,0 +1,167 @@
+/*
+ * Copyright 2016 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.mongodb.benchmark.benchmarks;
+
+import com.mongodb.benchmark.framework.BenchmarkResult;
+import com.mongodb.benchmark.framework.BenchmarkRunner;
+import com.mongodb.benchmark.framework.TextBasedBenchmarkResultWriter;
+import com.mongodb.client.MongoDatabase;
+import com.mongodb.client.gridfs.GridFSBucket;
+import com.mongodb.client.gridfs.GridFSBuckets;
+
+import java.io.File;
+import java.io.FileOutputStream;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.ExecutorService;
+import java.util.concurrent.Executors;
+import java.util.concurrent.TimeUnit;
+
+public class GridFSExportBenchmark extends AbstractMongoBenchmark {
```

Comment:
Suggest all benchmarks names are aligned to the `getName()` value.

Reply:
Done